templates/al2/runtime/bootstrap.sh

```diff
@@ -643,6 +643,8 @@ if [ -x "${BOOTSTRAP_GPU_HELPER}" ]; then
   log "INFO: completed GPU bootstrap helper!"
 fi
 
+set-nvidia-clocks
```

Comment:
ideally this would happen in the GPU bootstrap helper above; but it could just be a separate systemd unit too, I think? doesn't really matter when this happens, as long as it happens prior to a GPU pod being started

Reply:
that's true, i thought gpu bootstrap might be a dependency but in hindsight its just the container runtime initialization. should be able to just reuse the same service unit as the al23 template